bCNC/Pendant.py

```diff
@@ -89,26 +91,28 @@ def do_GET(self):
 				elif key=="cmd":
 					httpd.app.pendant.put(urllib.unquote(value))
 			#send empty response so browser does not generate errors
-			self.do_HEAD(200, "text/text")
+			self.do_HEAD(200, "text/text", cl=len(""))
 			self.wfile.write("")
 
 		elif page == "/state":
-			self.do_HEAD(200, content="text/text")
 			tmp = {}
-			for name in ["controller", "state", "pins", "color", "msg", "wx", "wy", "wz", "G"]:
+			for name in ["controller", "state", "pins", "color", "msg", "wx", "wy", "wz", "G", "OvFeed", "OvRapid", "OvSpindle"]:
 				tmp[name] = CNC.vars[name]
+			contentToSend = json.dumps(tmp)
+			self.do_HEAD(200, content="text/text", cl=len(contentToSend))
 			self.wfile.write(json.dumps(tmp))
```

Comment:
probably no need to call json.dumps() again, perhaps we can reuse contentToSend?

Reply:
Yes, i fix it, it's not optimized.